### DISCUSSION TITLE
Object Detection Result Analysis

### DISCUSSION THREAD

**Discussion by hardikdava** [roboflow/supervision#290] [Ideas]
Currently, we are only supporting `sv.ConfusionMatrix` and `sv.MeanAveragePrecision` in order to evaluate object detection model. We can extend object detection model evaluation by also providing summary of analysis or error analysis which is quite useful to interpret model results in production.

**Comment by SkalskiP** [comment]
Long-term, I agree! What would you like to see here? Any examples of tools that already support analysis like that?

Short-term, I would like to focus on tasks like this: https://github.com/roboflow/supervision/issues/292 and https://github.com/roboflow/supervision/issues/269 and polish evaluation API.
